### PR TITLE
Travis tests

### PR DESCRIPTION
**Pull request by ElenaPoduzova**
Add http tests to travis build

### CODE REVIEW

**Comment by JDIReporter** [comment]
[30] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[30] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[32] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[32] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[32] - Allure report on Netlify: https://5e3006b6c666a211662f66f5--jdi-dark.netlify.com

**Comment by JDIReporter** [comment]
[40] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[40] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[40] - Allure report on Netlify: https://5e30429eb26a2c304740b0c9--jdi-dark.netlify.com

**Comment by JDIReporter** [comment]
[42] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[42] - Could not save metadata Could not save metadata Could not save metadata

**Comment by JDIReporter** [comment]
[42] - Allure report on Netlify: https://5e304cc21ecc92d3a1c06a8c--jdi-dark.netlify.com

**Comment by JDIReporter** [comment]
[46] - https://transfer.sh/72H8o/jdi-dark-tests.tar.gz

**Comment by JDIReporter** [comment]
[46] - https://transfer.sh/EyRX9/jdi-dark-bdd-tests.tar.gz

**Comment by JDIReporter** [comment]
[46] - Allure report on Netlify: https://5e313ed9f9417bf965866491--jdi-dark.netlify.com

**Comment by JDIReporter** [comment]
[57] - https://transfer.sh/rfTuA/jdi-dark-bdd-tests.tar.gz

**Comment by JDIReporter** [comment]
[57] - https://transfer.sh/pgxwP/jdi-dark-tests.tar.gz

**Comment by JDIReporter** [comment]
[57] - Allure report on Netlify: https://5e319f46d4dbad7d927081af--jdi-dark.netlify.com

File: jdi-dark-tests/pom.xml [hunk 1]
```diff
@@ -22,13 +25,19 @@
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>6.11</version>
+            <version>7.0.0-beta1</version>
```
[the remark this answers is not in the document]
Comment (ElenaPoduzova) [reply jdi-dark-tests/pom.xml]: Fixed, thanks.

**Pull request merged**